Wikipedia/Code/WMFArticleListTableViewController.m

```diff
@@ -111,15 +111,21 @@ - (nullable UIViewController *)previewingContext:(id<UIViewControllerPreviewing>
     }
     [[PiwikTracker wmf_configuredInstance] wmf_logActionPreviewInContext:self contentType:contentType];
 
-    if (self.delegate) {
-        return [self.delegate listViewController:self viewControllerForPreviewingArticleURL:url];
-    } else {
-        return [[WMFArticleViewController alloc] initWithArticleURL:url dataStore:self.dataStore];
+    UIViewController *vc = self.delegate ?
+        [self.delegate listViewController:self viewControllerForPreviewingArticleURL:url] : [[WMFArticleViewController alloc] initWithArticleURL:url dataStore:self.dataStore];
+    
+    if ([[vc class] conformsToProtocol:@protocol(WMFArticlePreviewingActionsDelegate)]){
```

Comment:
@coreyfloyd Realized the new previewing actions delegate only needed to be set if we're *actually peeking a view controller*. Was much simpler this way too - i.e. we don't have to set it everywhere we were init'ing an article view controller.

Only needed to be set it in 3 places - here for lists, and in explore vc and article vc (for link peeking).